### ISSUE TITLE
"This element does not support ARIA roles, states, and properties" for noscript

### ISSUE THREAD

**Issue by calebegg** [GoogleChrome/accessibility-developer-tools#211]
The noscript tag gives this warning (AX_ARIA_12) if it has aria-hidden. This doesn't make sense, as it's (sometimes) a rendered node. (We are including it so that chromevox doesn't read the contents of the tag, because it does by default, even with JS enabled).

**Comment by RickSBrown** [comment]
@calebegg it's based on this: http://www.w3.org/TR/html-aria/#document-conformance-requirements-for-use-of-aria-attributes-in-html

There you will see that for `noscript`, in the column "ARIA roles, states and properties which may be used"
>
 "No role or aria* attributes"

**Comment by stevefaulkner** [comment]
@calebegg thats a bug in chromevox and needs to be reported, I suggest filing a bug http://crbug.com|crbug.com - tag with UI-Accessibility
